app/views/results/_outgoings_table.html.slim

```diff
@@ -1,4 +1,4 @@
-- if @early_eligibility_selection == "gross"
+- if @early_eligibility_selection == "gross" || @early_assessment == "gross_income"
```

Comment:
same as above, a comment to remind us to remove it, we could add the flag as a comment so when searching this will be returned